### DISCUSSION TITLE
<title>Unusual Traffic on GitHub hosted website

### DISCUSSION THREAD

**Discussion by isam2003** [community/community#110084] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello:  I observe some unusual activity on my website hosted on GitHub.  

A 240%+ increase in new users from a specific country with first user sources that I have never heard of before.  
<img width="84" alt="Screenshot 2024-02-25 064620" src="https://github.com/community/community/assets/49875705/15c11e0c-69d7-4f1c-a1f0-a965fc9b3fbb">
Is there a way to turn on geo-blocking rules on a website hosted on GitHub?  Thanks in advance.

**Comment by Akash1134** [comment]
Hi @isam2003 , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
